solid-terms.ttl

```diff
@@ -182,6 +184,15 @@ solid:oidcIssuer
     rdfs:label "OIDC issuer"@en ;
     rdfs:subPropertyOf <http://openid.net/specs/connect/1.0/issuer> .
 
+solid:owner
+    a rdf:Property ;
+    dc:issued "2021-05-19"^^xsd:date ;
+    rdfs:comment "A person or social entity that is considered to have control, rights, and responsibilities over a data storage."@en ;
+    rdfs:isDefinedBy <http://www.w3.org/ns/solid/terms#> ;
+    rdfs:domain rdfs:Resource ;
```

Comment:
Setting the domain to rdfs:Resource is an information-free thing to do, as that is true of every property. Please remove the line. (It is like saying A sameAs A, or A a Thing, or A a Resource)

Reply:
@matthieubosquet I don't adding information-free triples to things to be good practice.   Is there a reason you wanted that triple in?

Reply:
@timbl at first, I was unsure about it and just followed @TallTed’s [suggestion](https://github.com/solid/specification/issues/265#issuecomment-870118129) who might have specific reasons to express.

For me, explicitly stating that the domain of a property is `rdfs:Resource` shows to speakers of the vocabulary that the context in which `solid:owner` can be used has been discussed and tells them unequivocally that this property can be used in statements where the subject is any valid RDF subject node.

Reply:
By definition all classes are subclasses of rdfs:Resource and so that context is always present.

Reply:
@csarven I understand that stating: `ex:propertyX rdfs:domain rdfs:Resource .` is unnecessary to infer that subject nodes of statements where `ex:propertyX` is the predicate are instances of the class of everything, namely: `rdfs:Resource`.

But does stating `ex:propertyX a rdf:Property .` entail `ex:propertyX rdfs:domain rdfs:Resource .`?

I can’t really remember reading such axiom (starting from [rdf schema](https://www.w3.org/TR/rdf-schema/#ch_domain)).

So for sure it doesn’t entail further inferred triples, but I don’t think that not saying it is exactly the same thing as saying it (😄 I appreciate I might have easily missed something, I learn something new every time I read the RDF specs).

In any case, following that logic: asserting that something is an `rdf:Property` when it has an `rdfs:range` is redundant, so why assert it? To me, it’s simply nice to assert property type statements for consumers of a vocabulary, just because it makes the term definition more obvious and user friendly. When I read a vocabulary, I scan through it for specific info including type, domain and range statements.

Reply:
My feeling is that explicitly stating `rdfs:domain rdfs:Resource` makes it clear that this is what was intended by the ontology author(s), and that `rdfs:domain` was not accidentally left out (for intsance, where it would have been something more restricted than `rdfs:Resource`, had the author(s) included it).

In other words, I think that setting `rdfs:domain` to `rdfs:Resource` is _**NOT**_ an information-free thing to do, though it may seem so to a sophisticated observer (who knows that intentionally leaving the `rdfs:domain` declaration out has the same effect as making this explicit declaration) or upon casual observation.

Reply:
@timbl I applied your suggestion of removing the information-free `rdfs:domain` triple.

This should satisfied both changes requested cc @kjetilk.